about.html

```diff
@@ -11,54 +11,94 @@
 
 </head>
 
+<header>
+    <nav>
+    <img class="logo"  src="img/lambda-black.png" alt="Lambda Logo.">
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="#">Projects</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+    </nav>
+ 
+  </header>
+
 <body>
-    <div class="container about-page">
-     
-    About
+    <div class="container"></div>
```

Comment:
Be careful you are actually wrapping contents inside of your container. Notice how you have both your opening and closing `<div>` tags on the same line? This could throw you off later down the road when believe you are styling content that should be in the container, but actually is not.